config/tech-docs.yml

```diff
@@ -5,22 +5,24 @@ host: https://ministryofjustice.github.io/find-moj-data-user-guide
 
 # Header-related options
 show_govuk_logo: false
-service_name: Find MoJ data user guide
-service_link: /
-phase: INTERNAL
+service_name: Find MoJ data
+service_link: https://find-moj-data.service.justice.gov.uk/
+phase: PUBLIC BETA
 
 # Links to show on right-hand-side of header
 header_links:
-  GitHub: https://github.com/ministryofjustice/find-moj-data-user-guide
+  User Guide home: /
```

Comment:
```suggestion
  User guide home: /
```

Reply:
I think this user guide home link is a bit redundant but it can't hurt.

I was wondering if we were going to show a link back to the Find MoJ data service. However thinking about it, this will cause problems as all environments link to the same user guide so we would end up directing users from preprod to prod for example. So I don't think we should try to close the loop between the two sites.

Reply:
Currently I have added a link to prod in the top left hand corner to try to address the issues we saw at user testing. I could add a "Find MoJ data **(prod)**" bit to it to make the link to prod a bit clearer? I think it's worth it to have the link there, it came up so much at user testing. 

We don't currently have a large use case for users seeing different data in preprod, and data flows to all environments at the same time. It is a fair point though :/